### ISSUE TITLE
Outline for project-based courses blog post

### ISSUE THREAD

**Issue by ttimbers** [UBC-MDS/UBC-MDS.github.io-jekyll#29]
@mgelbart - FYI I have written down an outline for the project-based courses blog post: https://github.com/UBC-MDS/UBC-MDS.github.io/blob/project_post/_posts/2019-07-07-project-courses.md

Feedback welcome (am I missing anything?).

**Comment by ttimbers** [comment]
Decided to not discuss Capstone projects in detail in this post after discussion with @mgelbart. Will just link out to already existing materials.

**Comment by ttimbers** [comment]
@mgelbart - I have a rough draft of what was supposed to be the project based course blog done. It turns out I had a lot to say and it is FAR too long to be a blog post. Hence I propose the following:

1. I will copy everything I wrote to a new repo to be the basis of an educational leadership publication that I will ask the entire academic team to contribute to (if they desire). I hope to put it on archXiv and then submit it to the Journal of Statistical Education

2. I invite you to see if there is part (or parts) of it that could be extracted for a shorter blog post (I am OK with things being directly cut and paste from what I wrote as I believe it has no impact on plan 1 that I outlined above).

Thanks for your patience on my slow writing!
